### PR TITLE
new pingu dialogue

### PR DESCRIPTION
**Pull request by NotSteven1234**

### CODE REVIEW

**Comment by dawutheuwad** [comment]
hhmmMMmMMmmMMMMmmm

File: PinguUnfryer.py [hunk 1]
```diff
@@ -50,15 +74,18 @@ def Refry():
         print("Refrying Pingu, please wait...")
         time.sleep(2)
         print("Wait...")
-        time.sleep(2)
+         time.sleep(2)
```
Comment (dawutheuwad) [inline PinguUnfryer.py:77]: why you added a space here?

File: PinguUnfryer.py [hunk 2]
```diff
@@ -28,6 +28,30 @@
 print(50*"#")
 
 #Defines all functions
+
+def PinguLogin():
+    print("Logging in... Please wait.")
+    time.sleep(2)
```
Comment (dawutheuwad) [inline PinguUnfryer.py:34]: here, you forget about the banner

**Review by dawutheuwad — COMMENTED** [review]
It will be added possibly, but i need to fix the code a bit.

**Comment by NotSteven1234** [comment]
smh code is already perfect

jk

File: PinguUnfryer.py (hunk 2 above)
Comment (NotSteven1234) [reply PinguUnfryer.py:34]: smh that was rushed

File: PinguUnfryer.py (hunk 1 above)
Comment (NotSteven1234) [reply PinguUnfryer.py:77]: forgot lol

**Comment by dawutheuwad** [comment]
> smh code is already perfect
> 
> jk

you can always make another pull request :3

**Comment by NotSteven1234** [comment]
smh

**Pull request closed without merging**